dev-starts3.sh

```diff
@@ -134,19 +134,6 @@ ulimit -c unlimited
 #Set max open file limit to 10240
 ulimit -n 10240
 
-# Run m0dixinit
```

Comment:
Not needed?

Reply:
yes, it was removed after discussion with motr team